src/Workspaces/Remote/Core/ProjectSystem/WorkspaceProjectFactoryServiceDescriptor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem
+{
+    internal static class WorkspaceProjectFactoryServiceDescriptor
```

Comment:
I don't like this class name, but the Descriptors type we have at the parent namespace is very much right now the place where you get descriptors for our remote process descriptors. I want to probably rename that but there's extra work involved there.